src/components/Table/Table/Table.tsx

```diff
@@ -1,18 +1,22 @@
 import React, { FC, ReactElement } from "react";
 import cx from "classnames";
-import { VibeComponentProps } from "../../../types";
+import { SubIcon, VibeComponentProps } from "../../../types";
 import styles from "./Table.module.scss";
 import { ITableHeaderProps } from "../TableHeader/TableHeader";
 import { ITableBodyProps } from "../TableBody/TableBody";
 import { getTableRowLayoutStyles } from "./tableHelpers";
 import { getTestId } from "../../../tests/test-ids-utils";
 import { ComponentDefaultTestId } from "../../../tests/constants";
 
+export type TableContentSkeletonType = "long-text" | "medium-text" | "circle" | "rectangle";
+
 export interface ITableColumn {
   id: string;
   title: string;
   infoContent?: string;
   width?: number | { min: number; max: number };
+  icon?: SubIcon;
```

Comment:
I'm missing a code example. the Do and Don't don't show how it is used
![image](https://github.com/mondaycom/monday-ui-react-core/assets/6093192/5cfca99c-62e6-4008-bb55-d6940f0e637a)

Reply:
@shlomitc I'll talk to Yael for maybe adding a story for this case